### PR TITLE
Feature/rr 400 export batch to pdf

### PR DESCRIPTION
**Pull request by CaraHill**
https://govtnz.atlassian.net/browse/RR-400

This PR totally reworks the former PDF into an html page, and addresses most of the bugs flagged on the Jira ticket when this failed testing:
- date has been changed to Wellington local time, not UTC - the decision was made to not bother updating this on existing applications, but only worry about those going forward
- Total claimed rebates and Rebate amount have been removed as we are not calculating them
- Phone number has been updated so that it displays correctly
- The questions will no longer return "No answer" but will return the chosen answer
- All currency will now display appropriately, including commas if a big number
- The pdf now uses the location from the rebate form itself, not the rebate form's property
- Signatures should now display properly
- Authorised Council Officer is title case

Things that could not be addressed:
- proper page breaks - this is not working and as such sometimes cuts off in unfortunate places
- macron on Kapiti - I could not recreate this issue locally, in Chrome, Safari or Firefox

# How to test this change

- [ ] has tests
- [ ] at least one a reviewer ran the code

### CODE REVIEW

File: app/assets/stylesheets/pdf.css [hunk 1]
```diff
@@ -1,51 +1,80 @@
-@import url("https://fonts.googleapis.com/css?family=Fira+Sans");
+.page-break {
+  page-break-after: always;
+  page-break-inside: avoid;
+  /*overflow: hidden;*/
```
Comment (mischa-s) [inline app/assets/stylesheets/pdf.css]: is this having a effect now?

File: app/assets/stylesheets/pdf.css [hunk 2]
```diff
@@ -1,51 +1,80 @@
-@import url("https://fonts.googleapis.com/css?family=Fira+Sans");
+.page-break {
```
Comment (mischa-s) [inline app/assets/stylesheets/pdf.css]: is this having an effect now?

File: app/assets/stylesheets/pdf.css [hunk 3]
```diff
@@ -1,51 +1,80 @@
...
-  position: relative;
+  display: inline-block;
   height: 100px;
+  width: 100%;
 }
 
-.heading {
-  font-family: "Fira Bold", sans-serif;
+.heading h3 {
+  color: #ffffff;
```
Comment (mischa-s) [inline app/assets/stylesheets/pdf.css]: could this be var(--white)

File: app/assets/stylesheets/pdf.css [hunk 4]
```diff
@@ -1,51 +1,80 @@
...
-  margin: 30px 60px 0;
+.rebate-container-secondary {
+  float: none;
+  margin: 20px;
   position: relative;
 }
 
-.main-header {
+.heading {
   background-color: #000000;
```
Comment (mischa-s) [inline app/assets/stylesheets/pdf.css]: could this be var(--black)

File: app/assets/stylesheets/pdf.css [hunk 5]
```diff
@@ -1,51 +1,80 @@
...
 
-.container {
-  margin: 66px 60px 0;
-  font-family: "Fira Sans", sans-serif;
-  font-size: 16px;
-  color: #000000;
+.batch {
+  display: flex;
+  flex-direction: column;
+  float: none;
```
Comment (mischa-s) [inline app/assets/stylesheets/pdf.css]: is this having an effect?

File: app/assets/stylesheets/pdf.css [hunk 6]
```diff
@@ -1,51 +1,80 @@
...
+}
+
+.batch-container {
+  float: none;
```
Comment (mischa-s) [inline app/assets/stylesheets/pdf.css]: is this having an effect?

File: config/environments/staging.rb [hunk 7]
```diff
@@ -99,8 +99,12 @@
     tls: true
   }
 
+  config.time_zone = 'Wellington'
+
+  config.active_record.default_timezone = :local
+
   config.active_storage.service = :amazon
 
   # For pdfs on heroku
```
Comment (mischa-s) [inline config/environments/staging.rb]: we can probably just remove this?

File: app/assets/stylesheets/pdf.css (hunk 2 above)
Comment (CaraHill) [reply app/assets/stylesheets/pdf.css]: No, unfortunately it never seemed to work. If you have any ideas, I would love to hear them.

File: app/assets/stylesheets/pdf.css (hunk 3 above)
Comment (CaraHill) [reply app/assets/stylesheets/pdf.css]: Variables didn't work when it was a PDF, and I forgot to put them in now that it's an html. Thanks for noticing!

File: app/assets/stylesheets/pdf.css (hunk 2 above)
Comment (mischa-s) [reply app/assets/stylesheets/pdf.css]: nothing off the top - I can have a play with it - I'd say for now we should remove it otherwise its confusing

**Comment by CaraHill** [comment]
@mischa-s I think I've addressed all of your comments now, if you want to rereview. Thanks!

**Pull request merged**